### ISSUE TITLE
How to integrate a story through the scripts on messenger?

### ISSUE THREAD

**Issue by shivambats** [wit-ai/pywit#92]
Is there any way to integrate wit.ai stories to the facebook messenger?
I have a proper story in the wit.ai UI but I am having troubles integrating that to the messenger.

**Comment by shashankpr** [comment]
Hey ! There is an example program in the _examples_ folder that shows the integration between messenger and Wit.
If it is not clear, you can check by repo ! I have integrated my Wit stories with messenger. The Readme is not updated yet but you can ask me if you need any help : [https://github.com/shashankpr/chatbots](url).

**Comment by shivambats** [comment]
Hey @shashankpr , Thanks for the help, I have looked at the messenger.py file, but it didn't help me much. The issue is that I spent some time to make a story for static replies in wit.ai, There is a set of answer to every question. Basically I am integrating FAQs of my website in the bot. I made the story in such a way that the bot will give categories in quick replies, and then based on the category there will be question of that category, so there is no action going on at the back, there are some questions and those questions have the answers. I assumed it will be not too complex to integrate such thing with messenger. I even tried to export the data, a JSON file was downloaded. I am having a hard time understanding the concept of story, if we have to integrate the JSON. If there was an easier approach to integrate the story I have made.
